### PR TITLE
Pull Request try

### PR DESCRIPTION
**Pull request by uri-ustrell**
### Pull Request try

- I challenge you to comment and require some changes to keep this test alive!

### CODE REVIEW

**Comment by uri-ustrell** [comment]
> ### Pull Request try
> * I challenge you to comment and require some changes to keep this test alive!

giphy-downsized

**Comment by jbaldrich** [comment]
This code is bullshit 🤮

**Comment by uri-ustrell** [comment]
> This code is bullshit 🤮

Nice to hear from you, as always!

and now, try to reference one specific line to change or refactor

**Review by jbaldrich — CHANGES_REQUESTED** [review]
Consider line 116 as this adds nothing to the plugin

File: wpbp-security/README.txt [hunk 1]
```diff
@@ -111,4 +111,6 @@ Titles are optional, naturally.
 Markdown uses email style notation for blockquotes and I've been told:
 > Asterisks for *emphasis*. Double it up  for **strong**.
 
-`<?php code(); // goes in backticks ?>`
```
Comment (uri-ustrell) [inline wpbp-security/README.txt:114]: Blah blah

**Pull request closed without merging**